python/grass/temporal/temporal_vector_algebra.py

```diff
@@ -377,7 +378,8 @@ def set_temporal_extent_list(self, maplist, topolist=["EQUAL"], temporal="l"):
             map_new = self.generate_new_map(
                 base_map=map_i, bool_op="and", copy=True, rename=False, remove=True
             )
-            # Combine temporal and spatial extents of intermediate map with related maps.
+            # Combine temporal and spatial extents of intermediate map with related
+            # maps.
```

Comment:
```suggestion
            # Combine temporal and spatial extents of intermediate map with related maps
```

Without the period it is exactly 88 ;)